client/clientBean.go

```diff
@@ -44,7 +42,8 @@ import (
 )
 
 const (
-	ipPortRegex = `\b(?:(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\.){3}(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9]):[1-9]\d*\b`
+	ipPortRegex            = `\b(?:(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\.){3}(?:25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9]):[1-9]\d*\b`
```

Comment:
use something standard like this for IP validation https://golang.org/pkg/net/#ParseIP .. 

I am not sure if there ip:port validation, but the string can be split and worked with..

Reply:
actually I should remove this line. ipPortRegex is not being used anymore after I remove the old code.